unicorn_mode/samples/persistent/harness.c

```diff
@@ -8,7 +8,7 @@
 
    Run under AFL as follows:
 
-   $ cd <afl_path>/unicorn_mode/samples/persistent/
+   $ cd <afl_path>/afl->unicorn_mode/samples/persistent/
```

Comment:
really  "afl->unicorn_mode"?

Reply:
Hehe that one was a renaming bug. 🙃
It's in both harness files, too.